packages/core/src/routes/session.ts

```diff
@@ -336,6 +341,8 @@ export default function sessionRoutes<T extends AnonymousRouter>(router: T, prov
       );
 
       const id = await generateUserId();
+      ctx.log.userId = id;
```

Comment:
nit: No need to declare `id`, just simply
```typescript
ctx.log.userId = await generateUserId();
```

Reply:
cannot inline because there is another usage of `id`